scipy/stats/stats.py

```diff
@@ -633,10 +633,9 @@ def tvar(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
 
     """
     a = asarray(a)
-    a = a.astype(float).ravel()
+    a = a.astype(float)
     if limits is None:
-        n = len(a)
-        return a.var() * n / (n - 1.)
+        return np.ma.var(a, ddof=ddof, axis=axis)
```

Comment:
This change to `np.ma.var`, which I assume returns a masked array rather than an ndarray, doesn't seem justified. Why not just leave it at `a.var`? 

If you were aiming for consistency between `limits=None` and `limits` not None, I'd think regular arrays would be preferred.

Reply:
Got it, I added that for consistency b/c I was confused about the different calls.